### PR TITLE
python

### PR DESCRIPTION
**Pull request by sergioromcar**
test python

### CODE REVIEW

File: python/dirsearch.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+if sys.version_info < (3, 0):
+    sys.stdout.write("Sorry, dirsearch requires Python 3.x\n")
+    sys.exit(1)
+
+from lib.core import ArgumentParser
+from lib.controller import Controller
+from lib.output import CLIOutput, PrintOutput
+
+
+class Program(object):
```
Comment (sergioromcar-git) [inline python/dirsearch.py:18]: Add comment

File: python/dirsearch.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+        if self.arguments.quiet:
+            self.output = PrintOutput()
+        else:
+            self.output = CLIOutput()
+
+        self.controller = Controller(self.script_path, self.arguments, self.output)
+
+
+if __name__ == "__main__":
```
Comment (sergioromcar-git) [inline python/dirsearch.py:34]: add comment

File: python/dirsearch.py (hunk 1 above)
Comment (sergioromcar) [reply python/dirsearch.py:18]: fixed

File: python/dirsearch.py (hunk 2 above)
Comment (sergioromcar) [reply python/dirsearch.py:34]: fixed

File: python/dirsearch.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import os
+import sys
+
+if sys.version_info < (3, 0):
+    sys.stdout.write("Sorry, dirsearch requires Python 3.x\n")
+    sys.exit(1)
+
```
Comment (sergioromcar-git) [inline python/dirsearch.py:7]: add

Comment (sergioromcar) [reply python/dirsearch.py:7]: fixed

**Review by sergioromcar-git — APPROVED** [review]

**Pull request merged**